src/caw/caw_function.cc

```diff
@@ -150,9 +155,13 @@ CawFuncReply CawFunction::Profile(const Any& payload, KeyValueInterface& kv) {
   return {Status::OK, reply.SerializeAsString()};
 }
 
-CawFuncReply CawFunction::Stream(const Any& payload, KeyValueInterface& kv) {
-  // This function interracts with the kvstore
-  // to accomplish streaming functionality
+CawFuncReply CawFunction::Stream(const Any& payload,
+                                  const std::unordered_map<std::string, 
+                                    std::vector<std::function<void(const Any&)>> >&
+                                  current_streamers_) {
+  // This function will parse the payload and look for any hashtags 
```

Comment:
Just something to consider is you might not want current_streamers_ to be `const` since you will want to remove "dead" streamers/connections somewhere, and I assume this is where you will want to remove them.

Reply:
Really good catch, thanks!